src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -0,0 +1,250 @@
+        <PackageTags>ML.NET ML Machine Learning Data Science DataFrame Preparation DataView Analytics Exploration</PackageTags>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <!-- 
+      1591: Documentation warnings
+      NU5100: Warning that gets triggered because a .dll is not placed under lib folder on package. This is by design as we want MDAI to be under interactive-extensions folder.
+       -->
+        <NoWarn>$(NoWarn);1591;NU5100;MSML_GeneralName;MSML_ParameterLocalVarName;MSML_PrivateFieldName;MSML_TypeParamName;SA1028;SA1507;SX1101;MSML_NoInstanceInitializers</NoWarn>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddMDAIToInteractiveExtensionsFolder</TargetsForTfmSpecificContentInPackage>
+    </PropertyGroup>
+
+    <!-- The following properties are set to package M.D.A.Interactive with the M.D.A nuget package. If M.D.A.I undergoes TFM or dependency changes, we need to update the TargetFramework passed in below-->
+    <Target Name="AddMDAIToInteractiveExtensionsFolder">
+        <MSBuild Projects="./../Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj" Targets="_GetBuildOutputFilesWithTfm" Properties="TargetFramework=netcoreapp3.1">
+            <!-- Manually hardcoding the TargetFramework to netcoreapp3.1 as that is the one that MDAI targets -->
+            <Output TaskParameter="TargetOutputs" ItemName="_ItemsToIncludeForInteractive" />
+        </MSBuild>
+
+        <ItemGroup>
+            <_ItemsToIncludeForInteractive Update="@(_ItemsToIncludeForInteractive)" PackagePath="interactive-extensions/dotnet" />
+            <TfmSpecificPackageFile Include="@(_ItemsToIncludeForInteractive)" />
+        </ItemGroup>
+    </Target>
+
+    <ItemGroup>
+        <None Include="Converters.cs">
+            <DesignTime>True</DesignTime>
+            <AutoGen>True</AutoGen>
+            <DependentUpon>Converters.tt</DependentUpon>
+        </None>
+        <None Include="PrimitiveDataFrameColumn.BinaryOperators.cs">
+            <DesignTime>True</DesignTime>
+            <AutoGen>True</AutoGen>
+            <DependentUpon>PrimitiveDataFrameColumn.BinaryOperators.tt</DependentUpon>
+        </None>
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Apache.Arrow" Version="$(ApacheArrowVersion)" />
+        <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+        <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
+        <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
+        <PackageReference Include="System.Text.Encoding" Version="$(SystemTextEncodingVersion)" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Microsoft.ML.DataView\Microsoft.ML.DataView.csproj" />
```

Comment:
Just putting this out there for thought:

We didn't depend on the latest `Microsoft.ML.DataView` before. We depended on `1.0.0`. I wonder if it would be better if we didn't force this dependency to the latest version?

Reply:
Is there any unification or breaking change issues?  In general we've taken an aggressive update cadence with all packages in dotnet.  It helps to ensure folks are patched by default, since Nuget doesn't automatically lift transitive dependencies.

I think someone can still use an old Microsoft.ML with a new DataView right?

Reply:
In general, ML.NET makes heavy usage of `InternalsVisibleTo`.

However, I looked and I don't think that is true for `Microsoft.ML.DataView` because at one point we wanted to split it completely apart from `Microsoft.ML` (we even named it `Microsoft.Data.DataView` for a time). So I _think_ we should be OK with using a new `Microsoft.Data.DataView` with an older `Microsoft.ML`, but I don't think anyone tests it.